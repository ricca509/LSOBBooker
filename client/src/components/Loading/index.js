import React from 'react';

export default ({ loadingText = 'Loading...' }) => (
  <div>{ loadingText }</div>
);
