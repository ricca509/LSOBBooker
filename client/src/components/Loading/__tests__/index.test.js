import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Loading from '../';

describe('Loading component', () => {
  it('renders the default loadingText', () => {
    const wrapper = shallow(<Loading />)

    expect(toJSON(wrapper)).toMatchSnapshot();
  });

  it('renders the loadingText when passed as prop', () => {
    const wrapper = shallow(<Loading loadingText="test" />)

    expect(toJSON(wrapper)).toMatchSnapshot();
  })
});
