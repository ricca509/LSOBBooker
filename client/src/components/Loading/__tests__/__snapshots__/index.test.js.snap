exports[`Loading component renders the default loadingText 1`] = `
<div>
  Loading...
</div>
`;

exports[`Loading component renders the loadingText when passed as prop 1`] = `
<div>
  test
</div>
`;
